### DISCUSSION TITLE
Unable to Bootstrap FluxCD with Multiple Directories in a Single Repository Using FluxCD Provider

### DISCUSSION THREAD

**Discussion by imacq** [fluxcd/flux2#5091] [Q&A]
When attempting to bootstrap multiple FluxCD configurations into separate directories within the same Git repository, the following error occurs:

```
╷
│ Error: Bootstrap run error
│ 
│   with module.fluxcd_setup.flux_bootstrap_git.this,
│   on .terraform/modules/fluxcd_setup/main.tf line 11, in resource "flux_bootstrap_git" "this":
│   11: resource "flux_bootstrap_git" "this" {
│ 
│ sync path configuration ("test2/newdir2/") would overwrite path ("./test/newdir/") of existing Kustomization
╵
```

The setup involves deploying two kind clusters (prod and dev) and organizing them into different directories in the same Git repository (prod/, dev/, and stg/), but the bootstrap process results in a conflict in the sync_path configuration.

![image](https://github.com/user-attachments/assets/496e5bcf-493b-483e-b9ce-a6401f6b3cf0)

The second one fails with the above error.

A bootstrap is already running in cluster1. 

![image](https://github.com/user-attachments/assets/2c67cfa6-8fb0-4314-89c1-1d20414cc52a)

is it possible to achieve this?

Thanks

**Comment by stefanprodan** [comment]
You can run bootstrap only once per cluster. See here an example repo: https://github.com/fluxcd/flux2-kustomize-helm-example
